apps/roboshield/package.json

```diff
@@ -4,10 +4,14 @@
   "private": true,
   "scripts": {
     "build-server": "tsc --project tsconfig.server.json",
-    "build-next": "NEXT_BUILD=true pnpm build-server && NEXT_BUILD=true PAYLOAD_CONFIG_PATH=${PAYLOAD_CONFIG_PATH:-dist/payload.config.js} node dist/server.js",
+    "build-next": "next build",
     "build-payload": "payload build",
```

Comment:
Do we still have 3 build steps: `build-server`, `build-next` and `build-payload`?

Reply:
I think its unnecessary, we can have just one build-next step, it should take care everything